rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -541,6 +542,35 @@ do
   fi
 done
 
+# - 2.11.6 : Remove LDAP indexes
+STEP="Remove LDAP indexes"
+
+LDAP_REINDEX_NEEDED=0
+for index in activeTechniqueId cn container directiveId isDynamic isEnabled isModified isSystem machineId nodeGroupId nodeId ruleId software softwareId softwareVersion techniqueCategoryId techniqueId uuid
+do
+  if [ -e /var/rudder/ldap/openldap-data/${index}.bdb ]
+  then
+    rm -f /var/rudder/ldap/openldap-data/${index}.bdb
+    LDAP_REINDEX_NEEDED=1
+  fi
+done
+
+if [ ${LDAP_REINDEX_NEEDED} -ne 0 ]; then
+  echo -n "INFO: Unwanted OpenLDAP indexes have been detected and removed, reindexing..."
+
+  # First, remove indexes (excepted objectClass) from slapd.conf
+  TMPFILE=$(mktemp)
+  grep -Ev "^index|# Software search" /opt/rudder/etc/openldap/slapd.conf > ${TMPFILE}
+  sed -i "s%database monitor%index objectClass eq\n\ndatabase monitor%" ${TMPFILE}
+  mv ${TMPFILE} /opt/rudder/etc/openldap/slapd.conf
```

Comment:
Why not do everything inline, without a temporary file ?
Here is how to do with sed : 
sed -ie "/^index\|# Software search/d" slapd.conf